### PR TITLE
SMD-6: Reorganized .sln file and added test project

### PR DESCRIPTION
**Pull request by hbgwhite**
Reorganized solution to allow addition of new projects and added test project for administration

### CODE REVIEW

File: TabulateSmarterTestAdminPackage.Tests/Class1.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿using NUnit.Framework;
```
Comment (jtreuting) [inline TabulateSmarterTestAdminPackage.Tests/Class1.cs]: The default class got left in and should be removed.

**Review by jtreuting — CHANGES_REQUESTED** [review]
I don't see any actual tests, are those going to be included in a different PR.  Seems like they should be in this one?

**Comment by hbgwhite** [comment]
This pull request is completely project reorganization without any material changes to code. The purpose was to move the existing tabulator project down the hierarchy from the solution files so that additional projects (tests in this case) can be added.

File: TabulateSmarterTestAdminPackage.Tests/Class1.cs (hunk 1 above)
Comment (hbgwhite) [reply TabulateSmarterTestAdminPackage.Tests/Class1.cs]: removed.

**Review by jtreuting — APPROVED** [review]

**Pull request merged**